docs/BEARING.md

```
@@ -1,72 +1,39 @@
+    Phase 1 : Database-Change Lane : Generate/Plan/Rehearse : HOLMES Evidence
+    Phase 2 : Continuum Contract Lane : Wesley Compile : TTD/Echo Bundles
+    Phase 3 : Conformance Witnessing : Receipt-Family Proof : Shared Ownership Maps
+    Phase 4 : Full Multi-Repo Alignment : Cross-Host Codecs : Universal Realization
+```
 
-Current priority: make Wesley's Continuum role boring and inspectable without
-lying about what is already real.
+## Active Gravity
 
-That currently means:
+### 1. Continuum Alignment
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)